bugwarrior/services/github.py

```diff
@@ -352,9 +360,8 @@ def issues(self):
 
         issues = {}
         if self.involved_issues:
-            issues.update(
-                self.get_involved_issues(self.username)
-            )
+            involved = self.get_involved_issues(self.username)
+            issues.update(filter(self.filter_issues, involved.items()))
```

Comment:
We're throwing it away in the filter function, but not on `issues` itself. The `involved` intermediate is hanging around unnecessarily though *grumble*Python and function scoping*grumble* ;) .